lib/Importer/ONNXModelLoader.cpp

```diff
@@ -683,6 +683,58 @@ llvm::Error ONNXModelLoader::loadOperator(const ONNX_NAMESPACE::NodeProto &op) {
     return llvm::Error::success();
   }
 
+  if (typeName == "Pad") {
+    // Input
+    NodeValue input;
+    ASSIGN_VALUE_OR_RETURN_ERR(input,
+                               getNodeValueOrCreateConstantByName(op.input(0)));
+    auto inputDims = input.dims();
+    auto numDims = inputDims.size();
+
+    // Padding properties
+    unsigned_t mode = PaddingMode::CONSTANT; // default is constant
+    if (dict.count("mode")) {
+      std::string modeStr;
+      ASSIGN_VALUE_OR_RETURN_ERR(modeStr, loadStr(dict["mode"]));
+      if (modeStr == "constant") {
+        mode = PaddingMode::CONSTANT;
+      } else if (modeStr == "reflect") {
+        mode = PaddingMode::REFLECT;
+      } else if (modeStr == "edge") {
+        mode = PaddingMode::EDGE;
+      } else {
+        RETURN_ERR("Pad: Invalid mode");
+      }
+    }
+    float value = 0.f; // Default
+    if (dict.count("value")) {
+      ASSIGN_VALUE_OR_RETURN_ERR(value, loadFloat(dict["value"]));
+    }
+
+    // Pads are mandatory.
+    assert(dict.count("pads") && "Pad: The 'pads' property is mandatory");
```

Comment:
Thank you for replacing the `assert`s with Errors! Please use `RETURN_ERR_IF_NOT` for this one also.

Reply:
Fixed.